test/browser/getOwnerDocument.test.js

```diff
@@ -0,0 +1,39 @@
+import { createElement, render } from 'preact';
+import { setupScratch, teardown } from '../_util/helpers';
+
+/** @jsx createElement */
+
+describe('parentDom.ownerDocument', () => {
+	/** @type {HTMLDivElement} */
+	let scratch;
+
+	before(() => {
+		scratch = setupScratch();
+	});
+
+	after(() => {
+		teardown(scratch);
+	});
+
+	it('should reference the correct document from the parent node', () => {
+		let iframe = document.createElement('iframe');
+
+		scratch.appendChild(iframe);
+
+		let iframeDoc = iframe.contentDocument;
+
+		iframeDoc.write(
+			'<!DOCTYPE html><html><head></head><body><div></div></body></html>'
+		);
+
+		iframeDoc.close();
+
+		let iframeRootNode = iframeDoc.querySelector('div');
+
+		render('Hello', iframeRootNode);
+
+		expect(iframeRootNode.textContent).to.be.equal('Hello');
+		expect(iframeRootNode.ownerDocument).to.be.equal(iframeDoc);
```

Comment:
This has historically been rendered by your `iframeDoc.write` so this would've worked before as well. What wouldn't have worked is checking on `iframeRootNode.firstChild`

Reply:
I think i can get away with using `Text.ownerDocument` instead of rendering a div